README.md

```diff
@@ -66,21 +62,24 @@ Gosling.js is a declarative grammar for interactive (epi)genomics visualization
 
 The Gosling's key features compared to existing visualization libraries and grammars are as follows:
 
-- **Encoding/Data Scalability**: Gosling scales from whole genomes to single nucleotides via semantic zooming that updates visual encodings dynamically and by using the rendering and data access capabilities of [our HiGlass genomics visualization framework](http://higlass.io/).
+-   **Encoding/Data Scalability**: Gosling scales from whole genomes to single nucleotides via semantic zooming that updates visual encodings dynamically and by using the rendering and data access capabilities of [our HiGlass genomics visualization framework](http://higlass.io/).
 
-- **Expressiveness**: Gosling is designed to be expressive enough to generate pretty much any visualization of genome-mapped data, which we accomplished by basing the grammar on [our taxonomy of (epi)genomics data visualizations](https://onlinelibrary.wiley.com/doi/full/10.1111/cgf.13727).
+-   **Expressiveness**: Gosling is designed to be expressive enough to generate pretty much any visualization of genome-mapped data, which we accomplished by basing the grammar on [our taxonomy of (epi)genomics data visualizations](https://onlinelibrary.wiley.com/doi/full/10.1111/cgf.13727).
 
-- **Interactivity**: Gosling has intuitive and effective user interactions built in, including zooming and panning and [brushing and linking](https://infovis-wiki.net/wiki/Linking_and_Brushing). This enables flexible visualizations that cover a wide range of visual analysis scenarios, like overview + detail views with brushes or comparative views.
+-   **Interactivity**: Gosling has intuitive and effective user interactions built in, including zooming and panning and [brushing and linking](https://infovis-wiki.net/wiki/Linking_and_Brushing). This enables flexible visualizations that cover a wide range of visual analysis scenarios, like overview + detail views with brushes or comparative views.
 
 ## Learn More About Gosling
-- [Gosling.js Editor](https://gosling.js.org/)
-- [Community Examples](https://github.com/gosling-lang/gosling-examples)
-- [Documentation](https://github.com/gosling-lang/gosling-docs)
-- [Getting Started](https://github.com/gosling-lang/gosling-docs/blob/master/Getting-Started.md)
-- [Tutorial](https://github.com/gosling-lang/gosling-docs/blob/master/tutorials/Tutorial.md)
-- [Roadmap](https://github.com/gosling-lang/gosling.js/projects/1)
+
+-   [Home Page](https://gosling-lang.github.io/gosling-website/)
```

Comment:
What do you think of combining the four website-related links into a single link, i.e., "Home Page", "Examples", "Documentation", "Tutorials" into:

```markdown
- [Documentation](https://gosling-lang.github.io/gosling-website/)
```

Since we now have a well-structured website, I think we can just provide a single URL to the website.

Reply:
Sure